packages/esm-appointments-app/src/patient-appointments/patient-appointments-header.tsx

```diff
@@ -0,0 +1,40 @@
+import { useTranslation } from 'react-i18next';
+import React from 'react';
```

Comment:
nit: Stylistically, `import React from 'react';` should probably be the first line of any React component.

Reply:
I have some eslint mods in the pipeline that'll solve import order.